cdap-api/src/main/java/co/cask/cdap/api/templates/ApplicationTemplate.java

```diff
@@ -33,7 +33,7 @@
 public abstract class ApplicationTemplate<T> implements Application {
 
   /**
-   * Using the given configuration, configure an Adapter with the given configuration.
+   * Configure an Adapter with the given configuration.
    * This method is called when an adapter is created in order to define what Datasets, Streams, and
    * Plugins, and runtime arguments should be available to the adapter.
    *
```

Comment:
How about instead:
```
Called when an adapter is created in order to define what Datasets, Streams,
Plugins, and runtime arguments should be available to the adapter, as determined
by the given configuration.
```

(The duplicate "and" in the existing second sentence should be removed...if the first sentence is removed, this is the sentence that will be shown on method summary.)